BreadPlayer.Core/Common/PasswordStorage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace BreadPlayer.Core.Common
+{
+    public class PasswordStorage
+    {
+        // These constants may be changed without breaking existing hashes.
+        private const int SALT_BYTES = 24;
```

Comment:
Salt and Hash length should be equal. Refer to [this](https://www.codeproject.com/Articles/704865/Salted-Password-Hashing-Doing-it-Right).

So SHA512 Hash would have 64 bytes long Salt. (512bits / 8 = 64 bytes).